src/Searchable.php

```diff
@@ -152,11 +164,11 @@ public function datesToSearchable(array $array): array
         foreach ($this->getArrayableRelations() as $key => $value) {
```

Comment:
Just a small thing as I think it reads a bit better when used in `method_exists()`.

```suggestion
         foreach ($this->getArrayableRelations() as $key => $model) {
```